lint/tracing_debug_field/ui/main.rs

```diff
@@ -0,0 +1,11 @@
+use tracing::info;
+
+#[derive(Clone, Copy, Debug)]
+struct Wrapped(&'static str);
+
+fn main() {
+    let field = Wrapped("wrapped");
+    info!(field = tracing::field::debug(field), "using the function");
+    info!(field = ?field, "using the sigil");
+    info!(?field, "using shorthand with sigil");
+}
```

Comment:
what is this for? just testing?

Reply:
Yeah, this `ui/main.rs` produces some output when compiled. https://github.com/Manishearth/compiletest-rs then tests the output against `ui/main.stderr`.

This is to make sure that all the cases that I want the lint to catch are caught.